packages/angular_devkit/schematics/src/tree/static.ts

```diff
@@ -16,34 +16,41 @@ export function empty() {
 }
 
 export function branch(tree: Tree) {
```

Comment:
Please add TODOs to these functions to remove those VirtualTree hacks for 7.0

Reply:
Done.